server/sonar-db-dao/src/test/java/org/sonar/db/component/ComponentTesting.java

```diff
@@ -231,6 +240,7 @@ public static ComponentDto newProjectBranch(ComponentDto project, BranchDto bran
     checkArgument(project.qualifier().equals(Qualifiers.PROJECT));
     checkArgument(project.getMainBranchProjectUuid() == null);
     String branchName = branchDto.getKey();
+    String branchSeparator = branchDto.getBranchType() == PULL_REQUEST ? ":PULL_REQUEST:" : ":BRANCH:";
```

Comment:
Why not use the existing constants instead of hard-coding the separator strings?

Reply:
because we keep encapsulation of this implementation detail. ComponentTesting is used only for tests